documentation/staging/content/userguide/base-images/ocr-images.md

```diff
@@ -0,0 +1,403 @@
+        that links the software images
+        to your Oracle Single Sign-On login credentials.
+        This database is automatically checked when
+        you use `docker pull` to obtain images from OCR.
+
+   **Note**: This step is needed only once for each image name (not the tag level).
+   For example, if you accept the terms for `weblogic_cpu`
+   in the `middleware` repository, then
+   the acceptance applies to all versions of WebLogic CPU images.
+
+1. Provide Docker with credentials for accessing
+   the Oracle Container Registry. For example,
+   the following command will prompt for your SSO credentials:
+
+   ```shell
+   $ docker login container-registry.oracle.com
+   ```
+
+1. Use Docker to pull the desired image:
+
+   ```shell
+   $ docker pull container-registry.oracle.com/middleware/weblogic_cpu:12.2.1.4-generic-jdk8-ol8
+   ```
+
+1. Use Docker to display an inventory of your local image cache:
+
+   ```shell
+   $ docker images
+   ```
+
+1. If desired, then you can [inspect](#inspect-images) the content of the image.
+
+__Notes:__
+- If you are using a multi-node Kubernetes cluster,
+  or your Kubernetes cluster is remote from your locally created or pulled domain image,
+  then additional steps are usually required to ensure that your Kubernetes cluster can access the image.
+  See [Set up Kubernetes to access domain images](#set-up-kubernetes-to-access-domain-images).
+- The operator requires domain images to contain WebLogic Server 12.2.1.3.0 or later.
+  When using 12.2.1.3 images, the operator requires that
+  the images contain patches 29135930 and 27117282;
+  these patches are included in OCR 12.2.1.3 GA and CPU images.
+
+### Inspect images
+
+If you have local access to a WebLogic Server or Fusion Middleware Infrastructure image
+and the image originates from the Oracle Container Registry or
+was created using the WebLogic Image Tool,
+then you can use the following commands to determine their contents:
+
+* Check the WLS version:
+  ```text
+  $ docker run \
+    container-registry.oracle.com/middleware/weblogic_cpu:12.2.1.4-generic-jdk8-ol8 \
+    sh -c 'source $ORACLE_HOME/wlserver/server/bin/setWLSEnv.sh > /dev/null 2>&1 && java weblogic.version'
+  ```
+
+* Check the WLS patches:
+  ```text
+  $ docker run \
+  container-registry.oracle.com/middleware/weblogic_cpu:12.2.1.4-generic-jdk8-ol8 \
+  sh -c '$ORACLE_HOME/OPatch/opatch lspatches'
+  ```
+
+* If you have installed the [WebLogic Image Tool](https://github.com/oracle/weblogic-image-tool) (WIT),
+  then you can obtain useful version and patch information using the WIT
+  [inspect](https://oracle.github.io/weblogic-image-tool/userguide/tools/inspect-image/) command with
+  the `--patches` option.
+  For example:
+  ```
+  $ /tmp/imagetool/bin/imagetool.sh inspect \
+  --image=container-registry.oracle.com/middleware/weblogic_cpu:12.2.1.4-generic-jdk8-ol8 \
+  --patches
+  ```
+  ### Ensure you are using recently patched images
```

Comment:
This should not be indented - it's combined with the bullet above...

Reply:
Yes, thank you.